### ISSUE TITLE
Readme for Microsoft marketplace

### ISSUE THREAD

**Issue by jgsogo** [conan-io/conan-vs-extension#70]
I'm not sure if the text in the `README.md` (displayed in Github) should be the same as the text displayed in the marketplace, but we have to improve/change it for the marketplace ([here](https://github.com/conan-io/conan-vs-extension/blob/4f22503ae1942bdda787d0f02ab81e74085a0e18/vsix/publish-manifest.json#L7) we can use the file we want). Something more focused in Conan itself and then the extension.

**Comment by SSE4** [comment]
I agree we need to update README.md, currently it's more focused on developers. I believe we should move it to something like Contributing.md (create contributor's guide) and write entirely new README.md focused on end-users.
